homeassistant/components/rainmachine/__init__.py

```diff
@@ -79,9 +80,28 @@
 CONF_SOLARRAD = "solarrad"
 CONF_TEMPERATURE = "temperature"
 CONF_TIMESTAMP = "timestamp"
+CONF_UNITS = "units"
+CONF_VALUE = "value"
 CONF_WEATHER = "weather"
 CONF_WIND = "wind"
 
+# Config Validator for Flow Meter Data
+CV_FLOW_METER_VALID_UNITS = [
```

Comment:
You could speed this up by turning it into a set (since we're merely looking for presence).

Reply:
Also, let's alphabetize the entries.